babel/babel-plugin-cerebral-optimize-tags/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "babel-plugin-cerebral-optimize-tags",
+  "version": "1.0.0",
+  "description": "Babel plugin that optimize tags into constructor calls",
+  "main": "dist/index.js",
+  "scripts": {
+		"test": "jest",
+    "test:watch": "npm test -- --watch",
+    "build": "BABEL_ENV=production babel -d dist ./*.js",
+    "coverage": "npm test -- --coverage",
+    "prepublish": "npm run build"
+  },
+  "author": "Fabrice Weinberg <Fabrice@weinberg.me>",
+  "license": "MIT",
+  "repository": "https://github.com/cerebral/cerebral/babel/babel-plugin-cerebral-optimize-tags",
+  "devDependencies": {
+    "@cerebral/monorepo": "0.0.1",
+    "babel-cli": "^6.2.0",
```

Comment:
babel-cli and preset should be already in monorepo deps. just add necessary binaries if you need some